libkbfs/conflict_resolver.go

```diff
@@ -55,7 +54,8 @@ type ConflictResolver struct {
 	config           Config
 	fbo              *folderBranchOps
 	prepper          folderUpdatePrepper
-	log              logger.Logger
+	log              traceLogger
+	deferLog         traceLogger
```

Comment:
Can you please change the first `defer` in `conflictResolver.doResolve()` to use this when logging?